### ISSUE TITLE
old pagination results replaced on next page

### ISSUE THREAD

**Issue by alrazamc** [prescottprue/redux-firestore#249]
**What is the current behavior?**
I am paginating messages stored in sub collection, when loading next 20 messages, old messages in redux are replaced by new messages. I think it is due to because each result is stored at same path in redux store

**What is the expected behavior?**
should merge prev page result with next one

**Which version of redux-firestore are you using? What about other dependencies?**

**Which environments/browsers are affected by this issue? Did this work in previous versions or setups?**

**Minimal demo to reproduce issue (using [codesandbox](https://codesandbox.io/) or similar)**

**Comment by prescottprue** [comment]
Have you tried using `storeAs` with a unique key so that results are written to a different place of redux for each page?

The new v1 version stores all sub-collections at the top level by default, so this is hopefully not an issue in that version

**Comment by alrazamc** [comment]
Yes I am using storeAs (<parent-collection-id>-messages) to store at the top level but not for separate page
Currently I am storing and merging messages in react component's state. It would be nice to have a "mergeResults: true" option in query configuration

**Comment by RossCarlisle** [comment]
Just ran into this exact issue also, mergeResults boolean would be really sweet in query configuration instead of overriding the result and losing previous data. Trying to avoid storing into react components state like @alrazamc

**Comment by prescottprue** [comment]
Good to know there is interest here, thanks to all for posting
